src/vs/code/electron-sandbox/issue/issueReporterMain.ts

```diff
@@ -617,11 +583,11 @@ export class IssueReporter extends Disposable {
 						}, timeToWait * 1000);
 					}
 				}
-			}).catch(e => {
-				this.logSearchError(e);
+			}).catch(_ => {
+				// Ignore
```

Comment:
Can we log the error to console/main process instead, might be easy to miss an error the fetch method reports. Is there any utility on the electronservice to forward error messages to main process console ?